h_high: 69
h_low: 75
s_high: 75
s_low: 69
v_high: 69
v_low: 75

